csharp/src/Microsoft.ML.OnnxRuntime/NativeOnnxTensorMemory.cs

```diff
@@ -128,12 +166,19 @@ public override Span<T> GetSpan()
             Span<T> span = null;
             unsafe
             {
-                span = new Span<T>((void*)_dataBufferHandle, _elementCount);
+                span = new Span<T>((void*)_dataBufferPointer, _elementCount);
             }
 
             return span;
         }
 
+        public Memory<String> MemoryString()
```

Comment:
return Memory<string> instead of Memory<String> ?

Reply:
Can you expand the question? Sounds like "return x instead of x ?" :)

Reply:
Can we actually create a Memory<T> with T as string? I thought T must be a value type?

Reply:
From https://docs.microsoft.com/en-us/dotnet/api/system.memory-1.-ctor?view=netcore-2.2, it looks like it can be ref type.

It will throw exception if **"T is a reference type, and array is not an array of type T."**